boards/nordic/nrf54h20dk/nrf54h20dk_nrf54h20_cpuapp.yaml

```diff
@@ -14,3 +14,5 @@ flash: 512
 supported:
   - gpio
   - pwm
+  - spi
```

Comment:
This change is not related to watchdogs.

Reply:
I moved it to a separate commit. This change is needed to run following configuration in Twister:
https://github.com/zephyrproject-rtos/zephyr/pull/71677